frontend/src/translations/en.js

```
@@ -0,0 +1,178 @@
+  tasks: 'Tasks',
+  weekly: 'Weekly',
+  monthly: 'Monthly',
+  today: 'Today',
+  emailSync: 'Email Sync',
+  addTask: '+ Add Task',
+  profile: 'Profile',
+  logout: 'Logout',
+  changePassword: 'Change Password',
+  welcome: 'Welcome',
+  yourTasks: 'Your Tasks',
+  noTasks: 'No tasks yet',
```

Biome (2.4.7)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)